### PR TITLE
[purview-account] first release for `azure-purview-account`

### PR DESCRIPTION
**Pull request by msyyc**
autorest --version=3.4.5 --python --track2 --use=D:\dev3\autorest.python --use=@autorest\modelerfour@4.19.2 --python-sdks-folder=D:\dev3\azure-sdk-for-python\sdk --python-mode=update D:\dev3\azure-sdk-for-python\sdk\purview\azure-purview-account\swagger\README.md   --low-level-client

### CODE REVIEW

**Comment by iscai-msft** [comment]
@msyyc we check every readme.md, regardless of where it is, so it's kind of annoying.

You have to add the swagger readme path [here](https://github.com/Azure/azure-sdk-for-python/blob/main/eng/.docsettings.yml#L79) to have the CI ignore the check for this readme. You can just copy what storage does here and add it for purview

File: sdk/purview/azure-purview-account/README.md [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+
+credential = DefaultAzureCredential()
+client = PurviewAccountClient(endpoint="https://<my-account-name>.purview.azure.com", credential=credential)
+
+request = accounts.build_list_keys_request()
+
+response = client.send_request(request)
+try:
+    response.raise_for_status()
+    assert response.status_code == 200
```
Comment (iscai-msft) [inline sdk/purview/azure-purview-account/README.md:83]: @msyyc can you also include how you would list through the keys, and print out some information about the keys, i.e. they're names or something?

You'd probably have to cycle through `response.json()`

File: sdk/purview/azure-purview-account/tests/test_smoke.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+class PurviewAccountSmokeTest(PurviewAccountTest):
+
+    @PurviewAccountPowerShellPreparer()
+    def test_basic_smoke_test(self, purviewaccount_endpoint):
+        client = self.create_client(endpoint=purviewaccount_endpoint)
+        request = accounts.build_list_keys_request()
+        response = client.send_request(request)
+        response.raise_for_status()
+        assert response.status_code == 200
```
Comment (iscai-msft) [inline sdk/purview/azure-purview-account/tests/test_smoke.py:19]: similar to the readme, can you do some checks on the returned keys?

File: sdk/purview/azure-purview-account/tests/_util.py [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+from azure_devtools.scenario_tests import RecordingProcessor
+
+
+class PurviewAccountRecordingProcessor(RecordingProcessor):
+    def process_response(self, response):
+        response["body"]["string"] = '{"atlasKafkaPrimaryEndpoint":"000","atlasKafkaSecondaryEndpoint":"000"}'
```
Comment (msyyc) [inline sdk/purview/azure-purview-account/tests/_util.py:12]: avoid sensitive message leak

**Comment by msyyc** [comment]
Hi @iscai-msft The pylint report [error](https://dev.azure.com/azure-sdk/public/_build/results?buildId=1044183&view=logs&jobId=b70e5e73-bbb6-5567-0939-8415943fadb9&j=b70e5e73-bbb6-5567-0939-8415943fadb9&t=56d4e2e6-c43b-527c-bad7-234ebe7429dd). How to clear it?

File: eng/tox/allowed_pylint_failures.py [hunk 4]
```diff
@@ -54,6 +54,7 @@
     "azure-purview-nspkg",
     "azure-purview-scanning",
     "azure-purview-catalog",
+    "azure-purview-account",
```
Comment (msyyc) [inline eng/tox/allowed_pylint_failures.py:57]: allow pylint error

**Pull request closed without merging**